test/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable func-names */
+import { expect } from 'chai';
+import { ActivityBar, SideBarView, ViewControl, ViewItem } from 'vscode-extension-tester';
+import { KNativeConstants } from './common/contants';
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function extensionUIAssetsTest(): void {
+  describe("Verify extension's base assets available after install", () => {
+    let view: ViewControl;
+    let sideBar: SideBarView;
+
+    beforeEach(async function () {
+      this.timeout(4000);
+      view = new ActivityBar().getViewControl('Extensions');
+      sideBar = await view.openView();
+    });
+
+    it('Dependent Kubernetes extension is installed', async function () {
```

Comment:
Please update for readability. `it` "should install the Dependent Kubernetes extension"
Or something like that and the next one too.